engine/executor/dm/worker.go

```diff
@@ -263,9 +263,15 @@ func (w *dmWorker) tryUpdateStatus(ctx context.Context) error {
 // workerStatus gets worker status.
 func (w *dmWorker) workerStatus(ctx context.Context) frameModel.WorkerStatus {
 	var (
-		stage       = w.getStage()
-		code        frameModel.WorkerState
-		taskStatus  = &runtime.TaskStatus{Unit: w.workerType, Task: w.taskID, Stage: stage, CfgModRevision: w.cfgModRevision}
+		stage      = w.getStage()
+		code       frameModel.WorkerState
+		taskStatus = &runtime.TaskStatus{
+			Unit:             w.workerType,
+			Task:             w.taskID,
+			Stage:            stage,
+			StageUpdatedTime: time.Now(),
```

Comment:
Seems this assign tasks no effect

Reply:
I prefer to left this one, remove the write to StageUpdatedTime at jobmaster side

Reply:
We only update task status when task stage changed, so the StageUpdatedTime may not change for a long time, does this meet user expectations